lib/convection/control/stack.rb

```diff
@@ -105,7 +105,7 @@ def []=(key, value)
       end
 
       def get(*args)
-        @attributes.get(*args)
+        @attributes.get(*args, "** Warning: Could not resolve value [#{args[1]}] for stack [#{args[0]}] **")
```

Comment:
I think we should throw an exception here rather than adding some bogus text that will fail to evaluate in the CloudFormation controller.